packages/cli/src/cmds/beacon/options.ts

```diff
@@ -154,6 +154,13 @@ export const beaconPathsOptions: ICliCommandOptions<IBeaconPaths> = {
     description: "Path to output all logs to a persistent log file",
     type: "string",
   },
+
+  bootnodesFile: {
+    hidden: true,
+    description: "Bootnodes file path",
+    defaultDescription: defaultBeaconPaths.bootnodesFile,
```

Comment:
I guess we don't need default path for this as this is optional param

Reply:
Done on https://github.com/ChainSafe/lodestar/pull/3562/commits/b8b796e8218b08e6f82e73110ece98ab8e93e20e